src/main/java/com/openlattice/postgres/ResultSetAdapters.java

```diff
@@ -920,6 +922,27 @@ public static CollectionTemplateKey collectionTemplateKey( ResultSet rs ) throws
         return new CollectionTemplateKey( entitySetCollectionId, templateTypeId );
     }
 
+    public static OrganizationExternalDatabaseTable organizationAtlasTable( ResultSet rs ) throws SQLException {
+        UUID id = id(rs);
+        String name = name(rs);
+        String title = title(rs);
+        Optional<String> description = Optional.ofNullable( description( (rs) ) );
+        UUID organizationId = id(rs);
```

Comment:
I think this should be `organizationId(rs)`? (assuming that exists?)

Reply:
Will check

Reply:
It exists! Will fix